.cirrus.yml

```diff
@@ -19,3 +19,76 @@ task:
     cd test
     b2 -l250 warnings-as-errors=on warnings=all crypto=openssl stage_enum_if stage_dependencies include=/usr/local/include library-path=/usr/local/lib
     LD_LIBRARY_PATH=./dependencies ./enum_if
+
+build_and_store_wheels: &BUILD_AND_STORE_WHEELS
+  env:
+    CIBW_BUILD_VERBOSITY: 3
+    CIBW_REPAIR_WHEEL_COMMAND_MACOS: >
+      delocate-listdeps {wheel} &&
+      delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}
+
+  install_cibuildwheel_script:
+    - $PYTHON -m pip install cibuildwheel==2.12.0
+
+  run_cibuildwheel_script:
+    - cibuildwheel
+
+  wheels_artifacts:
+    path: "wheelhouse/*"
+
+build_macos_arm64_task:
+  name: Build macOS arm64 wheels.
+  trigger_type: manual
```

Comment:
I see, this is why these workflows don't run. did you mean to make these manual/

Reply:
I made it manual based on this comment:
https://github.com/arvidn/libtorrent/issues/7131#issuecomment-1310942640

The `trigger_type` seems to only support `manual`. There are also options to restrict it to monitor certain organizations, repositories, branches, etc. I haven't found an option that runs a task specifically on pull requests.

I want to be sure the setting doesn't end up triggering too many runs per day... wouldn't want to be the one that ruins the free M1 runner pool.

Reply:
I think if you just remove the trigger type field, and have it behave like the other cirrus jobs, I believe it would only run on pushes.
if it doesn't run on every PR, it's likely to break at some point without anyone noticing.

Reply:
Done.